controllers/namespacescope_controller.go

```diff
@@ -286,7 +285,6 @@ func (r *NamespaceScopeReconciler) DeleteRbacFromUnmanagedNamespace(instance *op
 
 // When delete NamespaceScope instance, cleanup all RBAC resources
 func (r *NamespaceScopeReconciler) DeleteAllRbac(instance *operatorv1.NamespaceScope) error {
-	instance = r.setDefaults(instance)
```

Comment:
We can't delete this line. Otherwise, the operator can't get `instance.Spec.ConfigmapName` if configmapName is omitted.

Reply:
good catch